duckduckgo/Android
# Add more frequent syncing

Asana Issue URL: https://app.asana.com/0/414730916066338/521243654781581

## Description
Fixes an issue where users upgrading to version 0.9.0 were not seeing the Privacy Grade. This PR:
• Runs the sync job on startup and every three hours.
• Removes unneeded post-processing of cached data to keep things light now that we are running this job more often
• Removes unused test resources

Refer to asana ticket for more detail.

## Steps to Test this PR:
1. Install version 0.8.0 of the app via the tag
1. Wait for data to download
1. Visit a few sites to make sure that data is downloaded and trackers are being blocked
1. Now install this branch as an upgrade
1. Ensure privacy grade appears
